packages/fiori/src/ProductSwitchItem.hbs

```diff
@@ -0,0 +1,43 @@
+{{#if targetSrc}}
```

Comment:
You should either
1. check for both targetSrc and target, because both don't have default value set
or 
2. set default value for target.

I think option 2 is better because that way we can reproduce the browser default behaviour out of the box.

Reply:
Added the default value of the target propety (default value of the target attr)